packages/layout/src/components/cards/components/button.tsx

```diff
@@ -1,17 +1,18 @@
 import React, { useEffect, useRef, MutableRefObject } from 'react';
-import { Flex, H3, H4 } from '@tpr/core';
-import styles from './button.module.scss';
-import { EditArrowUp, EditArrowDown } from './arrowButton';
+import { Flex, H3, H4, P } from '@tpr/core';
 import { ArrowDown, ArrowUp } from '@tpr/icons';
+import { EditArrowUp, EditArrowDown } from './arrowButton';
+import styles from './button.module.scss';
 
 interface UnderlinedButtonProps {
 	isOpen?: boolean;
 	onClick?: any;
 	tabIndex?: number;
 	isEditButton?: boolean;
 	onCollapseCallback?: () => void;
-	btnRef?: MutableRefObject<any>;
+	buttonRef?: MutableRefObject<any>;
 	isMainHeading?: boolean;
+	notHeading?: boolean;
```

Comment:
It would be better if this property were not a negative, eg `heading={true}` rather than `notHeading={false}` if you want a heading.